pypeit/core/wavecal/wvutils.py

```diff
@@ -281,6 +281,10 @@ def get_wave_grid(waves=None, gpms=None, wave_method='linear', iref=0, wave_grid
             wave_grid = np.power(10.0,newloglam)
 
         elif wave_method == 'iref': # Use the iref index wavelength array
+            msgs.info(f'iref for the list is set to {iref}')
+            msgs.info(f'The shape of the list is: {np.shape(waves)}')
+            msgs.info(f'shape of the first wave_grid in the list is: {np.shape(waves[iref])}')
+            msgs.info(f'The median wavelength in orders 0, 37, 74 is {np.median(waves[0])}, {np.median(waves[0])}, {np.median(waves[0])}')
```

Comment:
Is the last statement above specific to NIRSPEC?  I would either generalize or remove.

Reply:
It must be... I am happy to remove it.